pb/summary/summary.proto

```diff
@@ -45,6 +47,74 @@ message FailingTestSummary {
   string fail_test_link = 13;
 }
 
+// Represents the number of tests that have a flakiness at or above the threshold.
+// If a test is counted in this bucket for this threshold, it will not be counted
+// for buckets of a lower threshold.
+//
+// E.g. if a test has a flakiness of 50.0, and there are two buckets: threshold 20.0
+// and threshold 3.0, the test will be counted for the 20.0 bucket and not the 3.0
+// one, even though the flakiness is above both thresholds.
+message FlakyBucket {
+  // Minimum flakiness for the bucket
+  float threshold = 1;
+  // number of tests in the bucket
+  int32 tests = 2;
+}
+
+// Metrics about a specific test, i.e. passes, fails, total runs, etc.
+message TestInfo {
+  // Test name.
+  string name = 1;
+  // Test environment, i.e. staging-1.13
+  string env = 2;
+
+  int32 total_runs = 3;
```

Comment:
I think here `total_runs` should be `total_non_infra_runs`? We probably should also add comments about what does each counter mean here.

Reply:
Done